Makefile.work

```diff
@@ -527,7 +528,7 @@ endif
 
 	$(Q)$(DOCKER_RUN) \
 		$(SLAVE_IMAGE):$(SLAVE_TAG) \
-		bash -c "$(SONIC_BUILD_INSTRUCTION) $@;$(COLLECT_BUILD_VERSION); $(SLAVE_SHELL)"
+		bash -c "$(SONIC_BUILD_INSTRUCTION) $@ && $(COLLECT_BUILD_VERSION); $(SLAVE_SHELL)"
```

Comment:
COLLECT_BUILD_VERSION has already handled the return value issue, keep the return value as the same as SONIC_BUILD_INSTRUCTION.